src/search/landmarks/exploration.cc

```diff
@@ -147,158 +121,99 @@ void Exploration::build_unary_operators(const OperatorProxy &op) {
 */
 void Exploration::setup_exploration_queue(
     const State &state, const vector<FactPair> &excluded_props,
-    const unordered_set<int> &excluded_op_ids, bool use_h_max) {
+    const unordered_set<int> &excluded_op_ids) {
     prop_queue.clear();
 
-    for (size_t var_id = 0; var_id < propositions.size(); ++var_id) {
-        for (size_t value = 0; value < propositions[var_id].size(); ++value) {
-            ExProposition &prop = propositions[var_id][value];
-            prop.h_add_cost = -1;
+    for (auto &proposition : propositions) {
```

Comment:
`proposition` is not a fitting name, the first-level elements of `propositions` are nor propositions, only the second-level elements are. The fact that `prop` is an element of `proposition` makes it clear that there's something fishy here, I think. :-) Perhaps a name like `variable`, `var_propositions`, `propositions_for_variable` or similar instead.

Reply:
Hi Clemens, das ist als "resolved" markiert, aber die Namen sind noch dieselben. Sollten wir das noch besprechen?